model/commitqueue/commit_queue.go

```diff
@@ -83,3 +104,25 @@ func ClearAllCommitQueues() (int, error) {
 
 	return clearedCount, nil
 }
+
+type CommentData struct {
+	Modules []Module `json:"modules"`
+}
+
+func GetCommentData(comment string) (CommentData, error) {
+	data := CommentData{}
+
+	// trim comment until the first '{'
```

Comment:
I thought about more rigorous regex matching but figured the unmarshal will fail regardless.

Reply:
It seems really dicey to make humans put json in their comments. Is there a better user interface?

Reply:
I changed the user interface as you suggested to a more command line like interface.
We're now expecting something of the form: `evergreen merge --modules [module1:1234, module2:3456]`

Reply:
I'd do --module <one> --module <two>, to avoid needing to parse out brackets and whatever..